arbeitszeiterfassung_s4a/arbeitszeiterfassung_s4a/employee_checkin/employee_checkin.py

```diff
@@ -41,3 +41,26 @@ def create_working_time_log(doc, event):
 			},
 		)
 		working_time.save()
+
+# For logged in employees at midnight, create a checkin of type OUT at 23:59:59 and a checkin of type IN at 00:00:00
+def switch_checkins_at_midnight():
```

Comment:
```suggestion
def switch_checkins_at_midnight():
	"""Create an Employee Checkin of type 'OUT' at 23:59:59 and one of type 'IN' at 00:00:00"""
```

The docstring usually goes inside the function